api_specifications/evidence-api-0.1.0.yaml

```diff
@@ -64,9 +63,35 @@ paths:
           content:
             '*/*':
               schema:
-                $ref: >-
-                  https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/ErrorResponse
-
+                $ref: 'https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/ErrorResponse'
+  /artifacts/v1/_search:
+    post:
+      tags:
+        - Evidence
+      summary: Search for artifacts
+      description: |
+        API used to search  artifacts using search criteria
+      requestBody:
+        description: Search criteria for artifcat + RequestInfo meta data.
+        content:
+          '*/*':
+            schema:
+              $ref: '#/components/schemas/EvidenceSearchRequest'
+        required: true
+      responses:
+        '201':
```

Comment:
change response code to 200. only when creating a new object should we return 201, else we should return 200, as an indication of success. Do make this change in UPDATE API as well

Reply:
status code "200" for UPDATE API As well. This is not done yet.